aten/src/ATen/test/cpu_generator_test.cpp

```diff
@@ -54,19 +54,19 @@ TEST(CPUGenerator, TestMultithreadingGetEngineOperator) {
   auto gen2 = at::detail::createCPUGenerator();
   {
     std::lock_guard<std::mutex> lock(gen1->mutex_);
-    gen2 = gen1->clone(); // capture the current state of default generator
+    gen2 = Generator(gen1->clone()); // capture the current state of default generator
```

Comment:
Why is the `Generator()` constructor call needed here?

Reply:
Because `clone()` is a method of `at::GeneratorImpl` and returns `std::shared_ptr<at::GeneratorImpl>` and c-tor `at::Generator(std::shared_ptr<at::GeneratorImpl>)` is explicit

Reply:
Maybe there should be a `Generator::clone()` for this case?

Reply:
added